GVRf/Framework/framework/src/main/jni/engine/renderer/renderer.cpp

```diff
@@ -748,14 +751,36 @@ void Renderer::renderMaterialShader(RenderState& rstate, RenderData* render_data
              shader_manager->getErrorShader()->render(&rstate, render_data, curr_material);
              return;
          }
-        shader->render(&rstate, render_data, curr_material);
+         if ((render_data->draw_mode() == GL_LINE_STRIP) ||
+             (render_data->draw_mode() == GL_LINES) ||
+             (render_data->draw_mode() == GL_LINE_LOOP)) {
+             if (curr_material->hasUniform("line_width")) {
+                 float lineWidth = curr_material->getFloat("line_width");
```

Comment:
How about documenting "line_width" somewhere? Maybe a dedicated method in GVRMaterial?